src/EnergyPlus/FuelCellElectricGenerator.hh

```diff
@@ -574,27 +574,25 @@ namespace FuelCellElectricGenerator {
 
         void FigureAirEnthalpy(EnergyPlusData &state, Real64 FluidTemp, Real64 &Hair);
 
-        void FigureFuelHeatCap(EnergyPlusData &state, Real64 FluidTemp, Real64 &Cp);
+        void FigureFuelHeatCap(EnergyPlusData &state, Real64 FluidTemp, Real64 &Cp) const;
```

Comment:
For a class member method, adding the const attribute tells the compiler that this function will access member variables on the class instance, but does not modify them.  It is basically making the `this->` instance "read-only".  And so therefore it can only call other methods on the class if they themselves are also `const` methods.  Great question!

And I'm not sure if it impacts performance at all.  Surely not in a significant way in the context of E+.  Just helps make sure the intent of the function is known to users of it.

Reply:
It almost certainly will not affect performance, but besides documentation to let users know that the function will not modify the object, it also enables the function to be used on a const object

```cpp
const SomeThing st;
// can only call `const` member functions on st.
```